templates_test/singleton/boil_suites_test.tpl

```diff
@@ -336,5 +336,4 @@ func TestUpsert(t *testing.T) {
   t.Run("{{$tableName}}", test{{$tableName}}Upsert)
   {{end -}}
   {{- end -}}
-}
-{{- end -}}
\ No newline at end of file
+}
```

Comment:
Well, finally I found the required parameter in VS Code. Now everything should be fine. I hope =)